pyspectral/near_infrared_reflectance.py

```diff
@@ -252,7 +254,9 @@ def reflectance_from_tbs(self, sun_zenith, tb_near_ir, tb_thermal, **kwargs):
         self._rad3x_t11 = self.tb2radiance(tb_therm, lut=lut)['radiance']
         thermal_emiss_one = self._rad3x_t11 * self.rsr_integral
 
-        l_nir = self.tb2radiance(tb_nir, lut=lut)['radiance'] * self.rsr_integral
+        l_nir = self.tb2radiance(tb_nir, lut=lut)['radiance']
+        self._rad3x = l_nir.copy()
+        l_nir *= self.rsr_integral
```

Comment:
Could you explain the motivation here ?

Reply:
The night side needs to be set with the unmodified radiances. The multiplication with the RSR integral breaks changes the data to be used in some mystical way I didn't go through.

Reply:
Understood, thanks.